apps/condo/domains/notification/schema/SendMessageService.test.js

```diff
@@ -8,7 +8,8 @@ const { MESSAGE_SENDING_STATUS, MESSAGE_RESENDING_STATUS, MESSAGE_DELIVERED_STAT
 const sleep = async (time) => (new Promise((resolve => {
     setTimeout(resolve, time)
 })))
-
+// https://github.com/facebook/jest/issues/11500
+jest.setTimeout(30000)
```

Comment:
it's better to move it inside https://github.com/open-condo-software/condo/blob/master/apps/condo/jest.setup.js